server/db.mjs

```diff
@@ -16,25 +22,25 @@ export const addSpecies = async ({ name, scientific_name, code, added }) =>
     "INSERT INTO species(name, scientific_name, code, added) VALUES($1, $2, $3, $4)",
     [name, scientific_name, code, added],
   );
-// delete species
-export const deleteSpecies = (speciesId) =>
-  db.none("DELETE FROM species WHERE species_id = ${speciesId}", { speciesId });
+// delete species when i understand cascade
```

Comment:
cascading you often setup when you create the table as its a part of creating the foreign key. We can totally talk about this!